### DISCUSSION TITLE
Pass params from browser to next/link urls

### DISCUSSION THREAD

**Discussion by jaybytez** [vercel/next.js#39874] [Help]
We have a function in our non-next.js applications that is an onclick event handler and takes all the browser params and adds them onto any of the urls that have been clicked on a page to ensure tracking parameters are passed onto the url that was clicked.

This same function does not work for us right now in Next.js, the jquery/js onclick event handler adds the params but some part of the next/link lifecycle strips them off.

Is there a clean way to globally (all clickable links) add browser params onto links made with next/link?

**Comment by thanhtutzaw** [comment]
You means dynamic routes ?
I wish [this](https://stackoverflow.com/questions/43181606/nextjs-url-params-like-react-router) will helped
